src/node_kernels/TKESSTIDDESNodeKernel.C

```diff
@@ -106,15 +106,15 @@ void TKESSTIDDESNodeKernel::execute(
   omegaSq *= 2.0;
   Pk *= tvisc;
 
-  DblType rdl = visc/(density * kappa_ * kappa_ * dw * dw * stk::math::sqrt(0.5 * (sijSq + omegaSq) ));
+  DblType rdl = visc/(density * kappa_ * kappa_ * dw * dw * stk::math::sqrt(0.5 * (sijSq + omegaSq) ) + 1e-10);
```

Comment:
I am not entirely sure why this 1e-10 is here. It would be good to double-chek with @gantech if this is the right thing to do?

Reply:
It's an epsilon to prevent a divide by zero in the case that `omegaSq` and `sijSq` are both zero.

Reply:
@psakievich you should have the `epsilon` factor outside of `sqrt` now you are effectively adding `sqrt(1e-10)` which is `1e-5` probably not what you want

Reply:
Sorry ignore my previous comment... parsed the parenthesis wrong.